x-pack/platform/plugins/shared/streams/server/routes/content/route.ts

```diff
@@ -95,6 +125,9 @@ const importContentRoute = createServerRoute({
       name: z.string(),
     }),
     body: z.object({
+      include: z
+        .string()
+        .transform((value) => contentPackIncludedObjectsSchema.parse(JSON.parse(value))),
```

Comment:
I didn't find a way to avoid this since we pass stringified value in multipart data